lib/compiler/webpack-compiler.ts

```diff
@@ -75,6 +75,11 @@ export class WebpackCompiler {
     const compiler = webpack(webpackConfiguration);
 
     const afterCallback = (err: Error, stats: any) => {
+      if (err && stats === undefined) {
+        console.log(err);
+        return process.exit(1);
```

Comment:
Do we want to close the current process even if the `watch` mode is turned on?

Reply:
Since the err object is detached from the actual compiler result and issues an error catched by webpack which is most likely thrown by underlying tasks, it should be handled fatal.

In my test scenarios this were mostly 

```json
{
  errno: -13,
  code: 'EACCES',
}
```

Reply:
Can we add an inline comment then? To explain why this if () block is needed